Controller/MediaAdminController.php

```diff
@@ -77,10 +77,12 @@ public function listAction(Request $request = null)
         $datagrid->setValue('context', null, $context);
 
         // retrieve the main category for the tree view
-        $category = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        $rootCategory = $this->container->get('sonata.classification.manager.category')->getRootCategory($context);
+        // This should be safe as the root category has to exist for a given context but I do not like fatal errors
```

Comment:
Consider triggering a depreciation notice announcing an exception for next major if you think an exception would have been the right thing to do. See CONTRIBUTING.md